web-client/src/client/lib.rs

```diff
@@ -459,13 +458,10 @@ impl Client {
     /// Fetching blocks from the network is not yet available.
     #[wasm_bindgen(js_name = getBlockAt)]
     pub async fn get_block_at(&self, height: u32) -> Result<PlainBlockType, JsError> {
-        let block = self
-            .inner
-            .consensus_proxy()
-            .blockchain
-            .read()
-            .get_block_at(height, false)?;
-        Ok(serde_wasm_bindgen::to_value(&PlainBlock::from_block(&block))?.into())
+        let blockchain = self.inner.consensus_proxy().blockchain;
+        let blockchain = blockchain.read();
+        let block = blockchain.get_block_at(height, false)?;
```

Comment:
I'd have left it as it was

Reply:
https://github.com/nimiq/core-rs-albatross/pull/3550#discussion_r2588096374